### PR TITLE
[JENKINS-52386] Add preview for markup of a CredentialParameterDefinition

### PR DESCRIPTION
**Pull request by microscotch**
Add missing markup formater for CredentialParameterDefinition to fix [[JENKINS-52386] - credentials() parameter does not support HTML in description](https://issues.jenkins.io/browse/JENKINS-52386)

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by microscotch** [comment]
@Wadeck @daniel-beck (can't add reviewers, so mentioning you here)

File: src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java [hunk 1]
```diff
@@ -157,6 +157,21 @@ public void escapeAndMarkupFormatAreDoneCorrectly() throws Exception {
         collector.checkThat("parameters page should not leave param name unescaped", text, not(containsString("<param name>")));
         collector.checkThat("parameters page should mark up param description", text, containsString("<b>[</b>param description<b>]</b>"));
         collector.checkThat("parameters page should not leave param description unescaped", text, not(containsString("<param description>")));
+
+        page = wc.getPage(p, "configure");
+        form = page.getFormByName("config");
+        HtmlTextArea value = form.getTextAreaByName("parameter.description");
```
Comment (daniel-beck) [inline src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java:163]: This fails now.

I think the test improvement is not really necessary anymore, as this PR now only changes the presentation of the field on the UI. So this PR seems ready to go once the changes to this file are reverted.

Comment (microscotch) [reply src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java]: Hi @daniel-beck,

Test improvement has been reverted.

File: src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java [hunk 2]
```diff
@@ -156,7 +156,7 @@ public void escapeAndMarkupFormatAreDoneCorrectly() throws Exception {
         collector.checkThat("parameters page should escape param name", text, containsString("&lt;param name&gt;"));
         collector.checkThat("parameters page should not leave param name unescaped", text, not(containsString("<param name>")));
         collector.checkThat("parameters page should mark up param description", text, containsString("<b>[</b>param description<b>]</b>"));
-        collector.checkThat("parameters page should not leave param description unescaped", text, not(containsString("<param description>")));
+        collector.checkThat("parameters page should not leave param description unescaped", text, not(containsString("<param description>")));
```
Comment (daniel-beck) [inline src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java:159]: Please revert whitespace change.

**Review by daniel-beck — APPROVED** [review]

**Comment by microscotch** [comment]
Hi @raul-arabaolaza,

Any chance to get this PR merged ?

**Comment by microscotch** [comment]
Hi @daniel-beck,

What's missing to get this PR merged into master branch ?

Did I miss something in the process ?

**Review by jtnord — APPROVED** [review]

**Pull request merged**